docs/getting-started/index.html

```diff
@@ -4,19 +4,55 @@
 group: "navigation"
 ---
 <div>
-	<h2>Getting Started</h2>
-	<p>Below you will find various materials to give you an introduction to the W3C Web of Things and its technology.</p>
+	<h2>Introduction</h2>
+	<p>The W3C Web of Things (WoT) seeks to counter the fragmentation of the IoT, making it much easier to create 
+		applications without the need to master the disparate variety of IoT technologies and standards. 
```

Comment:
WoT defines an abstract architecture that is based on a set of requirements that were derived from use cases for multiple application domains. 
The architecture defines a basic conceptual framework that can be mapped onto a variety of concrete deployment scenarios and use cases.

Use cases and application domains include: 
- Consumer
- Industrial
- Transportation & Logistics
- Engineering and Construction
- Agriculture
- Healthcare
- Environment Monitoring
- Smart Cities
- Smart Buildings
- Connected Cars